### DISCUSSION TITLE
How to run a delayed Job

### DISCUSSION THREAD

**Discussion by dzvon** [apalis-dev/apalis#168] [Q&A] [answered]
Hi,

This is a cool project!

I would like to know if delayed jobs are currently supported or if there are any plans for them?

**Comment by geofmureithi** [comment, marked as answer]
You should be able to use schedule and chrono.
```rust
Utc::now() + chrono::Duration::seconds(i),
```
https://github.com/geofmureithi/apalis/blob/4c8a8254070f964caf8c77ccb47346d3d4c7f92d/examples/sqlite/src/main.rs#L11-L26
